labelbox/client.py

```diff
@@ -29,25 +29,24 @@ class Client:
     """ A Labelbox client. Contains info necessary for connecting to
     a Labelbox server (URL, authentication key). Provides functions for
     querying and creating top-level data objects (Projects, Datasets).
+    
     """
 
     def __init__(self,
                  api_key=None,
                  endpoint='https://api.labelbox.com/graphql'):
         """ Creates and initializes a Labelbox Client.
 
-            Logging is defaulted to level WARNING. To receive more verbose
-                output to console, update logging.level to the
-                appropriate level.
+        Logging is defaulted to level WARNING. To receive more verbose
+        output to console, update logging.level to the appropriate level.
 
-            >>> import logger
-            >>> logging.basicConfig(level = logging.INFO)
-            >>> client = Client("<APIKEY>")
+        >>> import logger
+        >>> logging.basicConfig(level = logging.INFO)
+        >>> client = Client("<APIKEY>")
 
         Args:
-            api_key (str): API key. If None, the key is obtained from
-                the "LABELBOX_API_KEY" environment variable.
-            endpoint (str): URL of the Labelbox server to connect to.
+            api_key (String): API key. If None, the key is obtained from the "LABELBOX_API_KEY" environment variable.
```

Comment:
You have been replacing "str" with `String` in most (all?) places... I don't think that's correct, generally speaking, as `str` is the proper name of a Python class representing a sequence of characters.

Reply:
Reverted all cases of String to back to str.